tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/LanguageRepoService.cs

```diff
@@ -45,6 +45,16 @@ public interface ILanguageRepoService
     /// <param name="packagePath">Absolute path to the package directory</param>
     /// <returns>CLI check response containing success/failure status and response message</returns>
     Task<CLICheckResponse> ValidateChangelogAsync(string packagePath, CancellationToken ct);
+
+    /// <summary>
+    /// SDK package paths should be the package name, as referred to by any scripts within the repo.
+    /// </summary>
+    /// <returns>The package name, suitable for passing to scripts that take a -PackagePath parameter. 
+    /// Some examples:
+    /// - In Go, this would be of the format "sdk/messaging/azservicebus"
+    /// - In Python, it would be "azure-servicebus".
+    /// </returns>
+    string GetSDKPackagePath(string repo, string packagePath);
```

Comment:
if this method returns azure-servicebus for Python then this should be renamed as GetSDKPackageName. How about JS?  For e.g. package name is @azure/template and directory name is template.